pwndbg/commands/context.py

```diff
@@ -352,9 +352,76 @@ def context(subcontext=None):
             out.flush()
 
 
+pwndbg.config.Parameter('show-compact-regs', False, 'whether to show a compact register view')
+pwndbg.config.Parameter('show-compact-regs-align', 20, 'the number of characters reserved for each register and value')
+pwndbg.config.Parameter('show-compact-regs-space', 4, 'the number of characters separating each register')
+
+
+def compact_regs(regs, width):
+    align = int(pwndbg.config.show_compact_regs_align)
+    space = int(pwndbg.config.show_compact_regs_space)
+    result = []
+
+    def calculate_padding(length):
+        return 0 if length % align == 0 else (align - (length % align))
+
+    def get_text_length(text):
+        '''Returns the text length while ignoring color-code escape sequences.'''
+        length = 0
+
+        escape_sequence = False
+        for char in text:
+            if escape_sequence:
+                escape_sequence = (char != 'm')
+            else:
+                escape_sequence = (char == '\033')
+                length += 0 if escape_sequence else 1
```

Comment:
Could be `length += int(escape_sequence)`, assuming its always `bool`.

(or even without the `int(...)` but that's not necessarily verbose?)